pkg/proxy/endpoints.go

```diff
@@ -445,7 +455,7 @@ func (ect *EndpointChangeTracker) endpointsToEndpointsMap(endpoints *v1.Endpoint
 				// Only supported with EndpointSlice API
 				zoneHints := sets.String{}
 
-				baseEndpointInfo := newBaseEndpointInfo(addr.IP, int(port.Port), isLocal, nil, isReady, isServing, isTerminating, zoneHints)
+				baseEndpointInfo := newBaseEndpointInfo(addr.IP, "", "", int(port.Port), isLocal, isReady, isServing, isTerminating, zoneHints)
```

Comment:
comment on why we pass "" for node and zone?  Can we use  addr.NodeName ?

Reply:
We can use `addr.NodeName`. This will be slightly different behavior than before since we were sending nil for the topology and the NodeName was not available for endpoints before.